torchvision/models/swin_transformer.py

```diff
@@ -126,7 +126,8 @@ def shifted_window_attention(
     qkv_bias: Optional[Tensor] = None,
     proj_bias: Optional[Tensor] = None,
     logit_scale: Optional[torch.Tensor] = None,
-):
+    training: bool = True,
```

Comment:
It is a bit sad that we can't put this parameter closer to the `dropout` ones (that would break BC).
Let's add a comment on top of `attention_dropout` saying something like 

```
# If you set the dropout parameters, don't forget to also set the training parameter below.
```

Reply:
I have added docstring for training. That would help